client/src/containers/blog-post/index.js

```diff
@@ -1,9 +1,20 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Layout from '../../components/Layout';
 import BlogPost from './BlogPost';
+import BlogPreviewFragment from './BlogPreviewFragment';
+import HeroFramgment from './HeroFragment';
 
-export default () => (
-  <Layout>
-    <BlogPost />
-  </Layout>
-);
+export default () => {
+  const [postDetails, setPostDetails] = useState({});
+
+  const handlePostLoaded = (post) => {
```

Comment:
This just creates an alias for `setPostDetails`. You can use `setPostDetails` instead of `handlePostLoaded` directly.